pages/blog/2020-summary.md

```diff
@@ -0,0 +1,929 @@
+    </td>
+    <td align="center">
+      Impressions
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      611<br/>
+      57 from payed marketing
+    </td>
+    <td align="center">
+      1253
+    </td>
+    <td align="center">
+      80095
+    </td>
+  </tr>
+</table>
+
+People that like your updates are most likely the ones that click on links. 75% (947) of likes converted into visits to our [website](http://asyncapi.com/). We took this data from Google Analytics, and we know that in 2019 we had only 137 visits from LinkedIn.
+
+The most performing update was the one about [partnership with Postman](https://www.linkedin.com/feed/update/urn:li:activity:6742099578591481856). It is LinkedIn, and it is all about business.
+
+The below charts show that data match reality. The highest peak in late March is related to our first [AsyncAPI Online Conference](https://www.youtube.com/watch?v=oMSzGc5bDr4). The rise of sponsored traffic is connected to our post-conference activities using the money we got from conference sponsors.
+
+<Figure
+  src="/img/posts/2020-summary/linkedin-impressions.webp"
+  caption="Figure 2: Impressions to updates in 2020"
+/>
+
+### Sponsored marketing
+
+This year we explored paid adverts on LinkedIn. We had some money left from sponsors of our AsyncAPI conference but zero experience with LinkedIn.
+
+Luckily we have a fantastic community. One of the community members, [Dan Weese](https://www.linkedin.com/in/daniel-weese-53b27410/), helped us here. He did not only put us in contact with marketing expert, [Charles Serdou](https://www.linkedin.com/in/sardou/), but also covered his wages. Thank you Dan :pray:
+
+This way, we covered only LinkedIn costs paying $1 074.08
+
+<Figure
+  src="/img/posts/2020-summary/linkedin-folowers.webp"
+  caption="Figure 3: Followers growth in 2020"
+/>
+
+Marketing was focused on increasing the number of followers. Comparing the results of sponsored marketing with followers' organic growth, one could say LinkedIn marketing was not so good. It was very good though. LinkedIn is just super expensive and charged a lot for every new follower.
+
+We should give it a second chance in 2021, but this time with sponsored updates.
+
+### What's next with LinkedIn
+
+The numbers show it is worth investing in the channel. We can reach a lot of people that are not reachable on other channels. So we will for sure continue what we do now, especially that this work doesn't require too much extra effort from us.
+
+I feel that probably some traffic coming from LinkedIn to our website is not adequately tracked in Google Analytics. Data from LinkedIn say we should notice 2x947 traffic. It might be related to people blocking tracking tools, and we will have to make sure to use other tracking methods, like putting identifiers in the links.
+
+Most important conclusion. If the AsyncAPI conference brought so much activity, it is evident that in 2021 we must organize another great AsyncAPI event.
```

Comment:
```suggestion
Most important conclusion: If the AsyncAPI conference brought so much activity, it is evident that in 2021 we must organize another great AsyncAPI event.
```

Reply:
Also, I'm not sure if this belongs here. This section is about LinkedIn and you're talking about the conference. Seems out of context.

Reply:
I think I'll remove entire sentence